.pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
@deshraj yeah, This file is for the pylint configuration. Like if we have to disable a pylint warning on complete project then we can do it using `disable` option within this file. Moreover, it is to make pylint checks according to our requirements, like `max-line-length` [here](https://github.com/Cloud-CV/EvalAI/pull/398/files#diff-d6d3d73911fd3ad6f39e4027f763b307R291) is set to `120`, whereas the default is `80`. There are many other features also which can be enabled or disabled.